src/tests/readytorun/JittedMethodsCountingTest/JittedMethodsCountingTest.cs

```diff
@@ -14,14 +14,16 @@ public class JittedMethodsCountingTest
     [Fact]
     public static int TestEntryPoint()
     {
-        // If DOTNET_ReadyToRun is disabled, then this test ought to be skipped.
-        if (!IsReadyToRunEnvSet())
+        // If either DOTNET_ReadyToRun or DOTNET_EnableHWIntrinsics are disabled
+        // (i.e. set to "0"), then this test ought to be skipped.
+        if (!IsReadyToRunEnabled() || !IsHardwareIntrinsicsEnabled())
         {
-            Console.WriteLine("\nThis test is only supported in ReadyToRun scenarios."
-                              + " Skipping...\n");
+            Console.WriteLine("\nThis test is only supported in ReadyToRun scenarios"
+                              + " with Hardware Intrinsics enabled. Skipping...\n");
             return 100;
         }
 
+        // This test is currently not compatible with the R2R-CG2 pipelines on Arm64.
```

Comment:
This comment is not very useful. It does not explain why it is not compatible. Good comments need to explain "why", not just "what".

Reply:
It seems that the problem with this configuration is no longer there. Will remove the check altogether.